docs/checklist.md

```diff
@@ -0,0 +1,42 @@
+# Checklist
+
+As a technopolitical project, Decidim needs several things to work. This is a non comprehensive list that serves as a general recommendation of what things you need to have it working with the best practices:
+
+## Technological
+
+1. Choose a **domain** for your application. Some typical names involve "Participation" or "Decision" conjugations.
+
+1. Choose which **languages** do you want for your application. In case that your language isn't supported you should translate it on [Crowdin](https://crowdin.com/project/decidim).
+
+1. Customize the **look and feel** (colors, pictures, fonts, etc).
```

Comment:
maybe I would add a link to [docs/customization/styles.md](https://github.com/decidim/decidim/blob/master/docs/customization/styles.md)

Reply:
:+1: